### PR TITLE
Fix: React rules shouldn't be in the base config

### PR DESCRIPTION
**Pull request by dgreene-r7**
Per the readme, the base config shouldn't lint React. Moved the react ruleset out of base and into index.

### CODE REVIEW

**Comment by tquetano-r7** [comment]
Based on this commit, pulling in the full config (index.js) would not pull in the react rules at all. If you can update the index.js to pull in the react rules as well, i can merge this.

File: package.json [hunk 1]
```diff
@@ -1,17 +1,11 @@
 {
-
-  "author": {
-    "name": "Ben Tesser",
-    "email": "btesser@rapid7.com"
-  },
+  "author": "Tony Quetano <tquetano@rapid7.com>",
```
Comment (tquetano-r7) [inline package.json]: Can you make this my proper external email? tony_quetano@rapid7.com

File: package.json [hunk 2]
```diff
@@ -1,17 +1,11 @@
...
   "bugs": {
     "url": "https://github.com/rapid7/esling-config-rapid7/issues"
   },
   "contributors": [
-    {
-      "name": "Ben Tesser",
-      "email": "btesser@rapid7.com"
-    }
+    "Tony Quetano <tquetano@rapid7.com>",
```
Comment (tquetano-r7) [inline package.json]: not sure this is valid ... shouldn't it be
{code}
"contributors": [
  {
    "name": "Tony Quetano",
    "email": tony_quetano@rapid7.com
  }, {
    "name": "David Greene",
    "email": "david_greene@rapid7.com"
  }
]
{code}

Comment (dgreene-r7) [inline package.json]: It is valid: https://docs.npmjs.com/files/package.json#people-fields-author-contributors

> Or you can shorten that all into a single string, and npm will parse it for you:
>
> `"Barney Rubble <b@rubble.com> (http://barnyrubble.tumblr.com/)"`

**Pull request merged**